src/tests/system/tests/test_netgroups.py

```diff
@@ -241,3 +243,131 @@ def test_netgroup__lookup_nested_groups_with_host_and_domain_values_present(
     result = client.tools.getent.netgroup("nested_group")
     assert result is not None
     assert expected in result.members
+
+
+@pytest.mark.importance("low")
+@pytest.mark.ticket(bz=802207)
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_netgroup__fully_qualified_names(client: Client, provider: GenericProvider):
+    """
+    :title: SSSD hangs when use fully qualified names set to true
+    :setup:
+        1. Configures the SSSD domain to use fully qualified names
+        2. User with the name "user-1" is created
+        3. A netgroup named "ng-1" is created and the previously added user "user-1" is added to this netgroup
+    :steps:
+        1. The netgroup_test function is executed three times to ensure
+            consistency and reliability of the configuration
+    :expectedresults:
+        1. Getent netgroup does not hang
+    :customerscenario: True
+    """
+    client.sssd.dom("test")["use_fully_qualified_names"] = "true"
+    user = provider.user("user-1").add()
+    provider.netgroup("ng-1").add().add_member(user=user)
+    client.sssd.start()
+
+    def netgroup_test():
```

Comment:
This is more of an opinion than anything else. I don't think adding this as a feature adds any added value. What's more, I think it makes it more difficult to read.